lib/ansible/config/manager.py

```diff
@@ -92,7 +92,16 @@ def ensure_type(value, value_type, origin=None):
 
         elif value_type == 'list':
             if isinstance(value, string_types):
-                value = [x.strip() for x in value.split(',')]
+                list_value = []
+                for item in value.split(','):
+                    item = item.strip()
+                    # in case this came as a 'string of quoted items' (ini, env var)
+                    for q in ('"', "'"):
```

Comment:
You could use `unquote` like string options to just remove the outer quotation marks. This removes all instances of it.

Reply:
only if i were smart ...

Reply:
was one of those days for me, I'm blaming it on monday :stuck_out_tongue_closed_eyes: